base/docs/basedocs.jl

```diff
@@ -2758,6 +2758,19 @@ See also [`setpropertyonce!`](@ref Base.setpropertyonce!) and [`setglobal!`](@re
 """
 setglobalonce!
 
+"""
+    declare_global(module::Module, name::Symbol, strong::Bool=false, [ty::Type])
+
+Declare the global `name` in module `module`.  If `ty` is given, declares a
+"strong" global, which cannot be replaced with a constant binding, otherwise
+declares a weak global.
+
+Does not update the current task's world age.
+
```

Comment:
Since this is impossible for any function to do, it seems odd to call out that this function doesn't do it
```suggestion
```

Reply:
The main reason to document Builtins is just for reference of argument order and meaning, not for users to use them